src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,78 @@
+package study;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final String DELIMITER = ",|:";
+    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private static final int ZERO = 0;
+    private static final int FIRST = 1;
+    private static final int SECOND = 2;
+
+    public int splitAndSum(String input) {
+        if (isNotValidValue(input)) {
+            return ZERO;
+        }
+        return sumIntArr(convertToNumberArray(splitString(input)));
+    }
+
+    private boolean isNotValidValue(String input) {
+        if (Objects.isNull(input) || "".equals(input)) {
+            return true;
+        }
+        return false;
```

Comment:
다음과 같이 줄여볼 수 있을 것 같아요.
```suggestion
        return Objects.isNull(input) || "".equals(input);
```